src/model/cpp_model.F90

```diff
@@ -2,7 +2,7 @@
 ! SPDX-License-Identifier: Apache-2.0
 !
 !> \file
-!> Fortran wrapper for aerosol models in C++
```

Comment:
I think these boilerplate comments are more trouble than they're worth. In most cases in the wild, they're out of date.

Reply:
I'm ok with removing them. I'll add this to #29